src/poliastro/tests/tests_twobody/test_perturbations.py

```diff
@@ -192,11 +193,13 @@ def test_atmospheric_drag():
     B = C_D * A / m
 
     # parameters of the atmosphere
-    rho0 = Earth.rho0.to(u.kg / u.km ** 3).value  # kg/km^3
-    H0 = Earth.H0.to(u.km).value
+    rho0 = rho0_earth.value  # kg/km^3
+    H0 = H0_earth.value
     tof = 100000  # s
 
-    dr_expected = -B * rho0 * np.exp(-(norm(r0) - R) / H0) * np.sqrt(k * norm(r0)) * tof
+    dr_expected = (
```

Comment:
Not sure why we were using the raw value before, doesn't this work using quantities?

Reply:
Finally solved this, sorry for being so late. I think units were avoid because when used, complexity inside `cowell`  makes it return a wrong result. Something similar happened some time ago in the `poliastro.maneuver` module